### PR TITLE
cassandra tracing documentation; resolves #455

### LINKED ISSUE #455: Document tracing Cassandra KVS

### PR DESCRIPTION
**Pull request by tpetracca**
@jboreiko there a good way to preview this?  I have no idea how markdown works and mostly just copied another page :P

### CODE REVIEW

**Comment by tpetracca** [comment]
Nvm about the preview.  Github makes it easy.

File: docs/source/configuration/enabling_cassandra_tracing.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+==============
+
+The ``atlasdb_tracing.prefs`` is a standard java properties file with 
+the following parameters:
+
+.. code:: properties
+
+    tracing_enabled: true           # self explanatory
+    trace_probability: 1.0          # the probability we trace an eligible query
+    min_duration_to_log_ms: 1000    # the minimum amount of time a traced query has to take to actually be logged
```
Comment (schlosna) [inline docs/source/configuration/enabling_cassandra_tracing.rst]: Probably worth noting that this is a postfilter in the current implementation, while probability and tables to trace are pre-query predicate filters

Comment (clockfort) [inline docs/source/configuration/enabling_cassandra_tracing.rst]: Current implementation and any possible implementation :-P

Also mention in general somewhere that this is expensive and causes extra write load.

**Comment by tpetracca** [comment]
Guys I broke atlasdb-tests-shared :P

File: docs/source/configuration/enabling_cassandra_tracing.rst (hunk 1 above)
Comment (schlosna) [inline docs/source/configuration/enabling_cassandra_tracing.rst]: You don't want to implement predicate push down tracing in C* 3.4+ a la https://github.com/thelastpickle/cassandra-zipkin-tracing/blob/master/src/main/java/com/thelastpickle/cassandra/tracing/ZipkinTracing.java ?

**Pull request merged**